HHS/Head-Start-TTADP
# [PROD] 0829 - TR Restructure and Alerts

## Description of change

* **TTAHUB-3247, TTAHUB-3246, TTAHUB-3322, TTAHUB-3196:** Restructure of both the Event and Session for role based field access and submission.

Add alerts to display on the top of the TR page. Also remove, update, and add a selection of new emails for the training reports.

## How to test

* **TTAHUB-3247, TTAHUB-3246, TTAHUB-3322, TTAHUB-3196:** Only select roles can complete and select certain aspects of the TR.

Confirm alerts appear. Emails are sent daily at 4 AM based on report dates but have been tested on lower environments.

## Issue(s)
* https://jira.acf.gov/browse/TTAHUB-3197
* https://jira.acf.gov/browse/TTAHUB-3199
* https://jira.acf.gov/browse/TTAHUB-3248
* https://jira.acf.gov/browse/TTAHUB-3198
* https://jira.acf.gov/browse/TTAHUB-3247
* https://jira.acf.gov/browse/TTAHUB-3246
* https://jira.acf.gov/browse/TTAHUB-3322
* https://jira.acf.gov/browse/TTAHUB-3196

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
